content/css/concepts/transform-functions/terms/translate/translate.md

```diff
@@ -13,24 +13,28 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Translates an element by one or more axes.
+The `translate` function moves an element along the X- and/or Y-axis.
 
 ## Syntax
 
-```css
-transform: translate(<value>);
+```pseudo
+/* With a single value */
+transform: translate(<X-value>);
+
+/* With both values */
+transform: translate(<X-value>, <Y-value>);
 ```
 
-where a required `<value>` can be one of the following:
+A required `<X-value>` or `<Y-value>` can be one of the following:
 
 - Length value: `100px`, `1.5em`
 - Percentage value: `25%`, `50%`
 
-**Note:** Providing only a single value will represent the horizontal axis. Providing a second comma separated value will represent the vertical axis. Values can be negative to translate elements in the opposite direction along the axes.
+Providing only a single value will represent the horizontal axis. Providing a second comma-separated value will represent the vertical axis. A positive value moves the element towards the right along the horizontal X-axis and down the vertical Y-axis. A negative value moves the element left along the horizontal X-axis and up the vertical Y-axis.
```

Comment:
```suggestion
A single value will represent only the horizontal axis. Providing a second comma-separated value will represent the vertical axis, as well. As as the direction goes:

* A positive value moves the element towards the right along the horizontal X-axis and down the vertical Y-axis (southeast). 
* A negative value moves the element left along the horizontal X-axis and up the vertical Y-axis (northwest).
```

Reply:
Is there more to this with respect to moving in a northeast or southwest direction?